corehq/apps/geospatial/routing_solvers/pulp.py

```diff
@@ -97,12 +99,20 @@ def get_decision_variables(x_dim, y_dim):
 
     @staticmethod
     def add_user_case_assignment_constraint(
-        lp_problem, decision_variables, user_count, case_count
+        lp_problem, decision_variables, user_count, case_count, min_cases=None, max_cases=None
     ):
-        # This constrain enforces cases being split equally between users.
-        max_constraint = int(case_count / user_count) + 1
+        # This constrain enforces the min/max amount of cases that could be assigned to each user,
+        # with the default being the cases split equally between users.
+        max_constraint = int(user_count / case_count) + 1
```

Comment:
For some reason I made a doozy and switched the `case_count` and `user_count` here (switched back [here](https://github.com/dimagi/commcare-hq/pull/34657/commits/91f5fd46be7283e096dc03e929be9680f9886efd))